templates/base-ubuntu-2004/Dockerfile.j2

```diff
@@ -24,8 +24,9 @@ RUN cat /etc/apt/sources.list.mirror-$(uname -m) > /etc/apt/sources.list
 
 RUN set -xeu && \
     ln -snf /usr/bin/bash /usr/bin/sh && \
-    install_packages busybox python3-pip supervisor xsltproc curl tree python-is-python3 openssh-client openssh-server \
-    krb5-user jsvc libssl1.1 && \
+    install_packages busybox python3-pip supervisor xsltproc curl tree python-is-python3 \
+    openssh-client openssh-server sqlite3 \
```

Comment:
Where is the use case for `sqlite3`?

Reply:
submitted in https://github.com/awesome-kyuubi/hadoop-testing/pull/87